juriscraper/opinions/united_states/state/colo.py

```diff
@@ -285,16 +282,22 @@ def _get_nature_of_suit(self):
     @staticmethod
     def cleanup_content(content):
         tree = html.fromstring(content)
```

Comment:
Yeah, I'd expect that too, but it does just give a falsey core_element.  I tested thoroughly, and got no errors, even when running sample_caller, which was throwing the errors that are currently causing coloctapp to fail.  Feel free to test it yourself, just don't dump "content" on a PDF or your terminal might explode (from experience) 

Sent from a phone

> On Aug 12, 2016, at 8:32 AM, Mike Lissner <notifications@github.com> wrote:
> 
> In juriscraper/opinions/united_states/state/colo.py:
> 
> > @@ -285,16 +282,22 @@ def _get_nature_of_suit(self):
> >      @staticmethod
> >      def cleanup_content(content):
> >          tree = html.fromstring(content)
> Does this really work this way? I mean, if you do html.fromstring then tree.xpath on a PDF, do you simply wind up with a falsy core_element value? I'd expect a crash, not just a failure to get data?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>